torch/nn/modules/linear.py

```diff
@@ -178,4 +179,78 @@ def extra_repr(self) -> str:
             self.in1_features, self.in2_features, self.out_features, self.bias is not None
         )
 
+
+class LazyLinear(LazyModuleMixin, Linear):
+    r"""A :class:`torch.nn.Linear` module with lazy initialization.
+
+    This module will initialize the `weight` and `bias` parameters after
+    the first call to `forward` is done. After that the module will
+    become a :class:`torch.nn.Linear` dropping all the lazy initialization
+    logic.
+
+    Be sure to run "dummy" forward once to initialize all parameters in case
```

Comment:
I don't understand what the punctuation is trying to get across.  Why is dummy quoted but real isn't?  Why is the first forward not double `` but the second one is?

Reply:
I think it should be consistent now, thanks!